packages/node_modules/@webex/internal-plugin-ediscovery/src/index.js

```diff
@@ -131,11 +131,9 @@ registerInternalPlugin('ediscovery', EDiscovery, {
           if (!object || !object.body) {
             return Promise.resolve();
           }
-          // the report id is needed in order to retrieve space information for activities with whiteboards so that
-          // the list of users in those spaces can be accessed and the content can be decrypted on their behalf
-          const reportId = object.options.resource.split('/')[1];
 
-          return Promise.all(object.body.map((item) => ctx.transform('decryptReportContent', {body: item}, reportId)));
+          // Always use the report url as this'll resolve correctly for remote reports
+          return Promise.all(object.body.map((item) => ctx.transform('decryptReportContent', {body: item}, object.options.uri)));
```

Comment:
This transform calls `getContentSummaryBySpaceId(..)`. By having one method it can always use a `uri` rather than choose between the previous method and `getRemoteContentSummaryBySpaceId(..)`.